zephyr/Kconfig

```diff
@@ -60,8 +60,6 @@ config INLINE_FUNCTIONS
 	bool
 	default y if CAVS_VERSION_2_5 || ACE
 	default n
-	depends on CAVS_VERSION_2_5
```

Comment:
This change should be squashed in previous commit (no 2 commit).

Reply:
sorry, should rebase instead of adding.

Reply:
@iuliana-prodan probably worth checking your platforms and enabling when tests pass, IIRC this was an old XCC issue so probably good for all modern targets.

Reply:
yes, definitely - I'll keep an eye on this :)